src/main/java/net/minestom/server/extras/vanilla/VanillaBlocks.java

```diff
@@ -0,0 +1,86 @@
+        BLOCKS[Block.STRIPPED_JUNGLE_LOG.getBlockId()] = new DAxisBlock(Block.STRIPPED_JUNGLE_LOG, "dv_stripped_jungle_log",  id++);
+        BLOCKS[Block.STRIPPED_ACACIA_LOG.getBlockId()] = new DAxisBlock(Block.STRIPPED_ACACIA_LOG, "dv_stripped_acacia_log",  id++);
+        BLOCKS[Block.STRIPPED_DARK_OAK_LOG.getBlockId()] = new DAxisBlock(Block.STRIPPED_DARK_OAK_LOG, "dv_stripped_dark_oak_log",  id++);
+        BLOCKS[Block.PURPUR_PILLAR.getBlockId()] = new DAxisBlock(Block.PURPUR_PILLAR, "dv_purpur_pillar",  id++);
+        BLOCKS[Block.QUARTZ_PILLAR.getBlockId()] = new DAxisBlock(Block.QUARTZ_PILLAR, "dv_quartz_pillar",  id++);
+        BLOCKS[Block.OAK_WOOD.getBlockId()] = new DAxisBlock(Block.OAK_WOOD, "dv_oak_wood",  id++);
+        BLOCKS[Block.SPRUCE_WOOD.getBlockId()] = new DAxisBlock(Block.SPRUCE_WOOD, "dv_spruce_wood",  id++);
+        BLOCKS[Block.BIRCH_WOOD.getBlockId()] = new DAxisBlock(Block.BIRCH_WOOD, "dv_birch_wood",  id++);
+        BLOCKS[Block.JUNGLE_WOOD.getBlockId()] = new DAxisBlock(Block.JUNGLE_WOOD, "dv_jungle_wood",  id++);
+        BLOCKS[Block.ACACIA_WOOD.getBlockId()] = new DAxisBlock(Block.ACACIA_WOOD, "dv_acacia_wood",  id++);
+        BLOCKS[Block.DARK_OAK_WOOD.getBlockId()] = new DAxisBlock(Block.DARK_OAK_WOOD, "dv_dark_oak_wood",  id++);
+        BLOCKS[Block.STRIPPED_OAK_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_OAK_WOOD, "dv_stripped_oak_wood",  id++);
+        BLOCKS[Block.STRIPPED_SPRUCE_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_SPRUCE_WOOD, "dv_stripped_spruce_wood",  id++);
+        BLOCKS[Block.STRIPPED_BIRCH_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_BIRCH_WOOD, "dv_stripped_birch_wood",  id++);
+        BLOCKS[Block.STRIPPED_JUNGLE_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_JUNGLE_WOOD, "dv_stripped_jungle_wood",  id++);
+        BLOCKS[Block.STRIPPED_ACACIA_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_ACACIA_WOOD, "dv_stripped_acacia_wood",  id++);
+        BLOCKS[Block.STRIPPED_DARK_OAK_WOOD.getBlockId()] = new DAxisBlock(Block.STRIPPED_DARK_OAK_WOOD, "dv_stripped_dark_oak_wood",  id++);
+        BLOCKS[Block.CRIMSON_STEM.getBlockId()] = new DAxisBlock(Block.CRIMSON_STEM, "dv_crimson_stem",  id++);
+        BLOCKS[Block.WARPED_STEM.getBlockId()] = new DAxisBlock(Block.WARPED_STEM, "dv_warped_stem",  id++);
+        BLOCKS[Block.STRIPPED_WARPED_STEM.getBlockId()] = new DAxisBlock(Block.STRIPPED_WARPED_STEM, "dv_stripped_warped_stem",  id++);
+        BLOCKS[Block.STRIPPED_CRIMSON_STEM.getBlockId()] = new DAxisBlock(Block.STRIPPED_CRIMSON_STEM, "dv_stripped_crimson_stem",  id++);
+
+        BLOCKS[Block.CHEST.getBlockId()] = new DChestBlock(Block.CHEST, "dv_chest",  id++);
+        BLOCKS[Block.TRAPPED_CHEST.getBlockId()] = new DChestBlock(Block.TRAPPED_CHEST, "dv_trapped_chest",  id++);
+
+        VANILLA_LAST_ID = (short) (id - 1);
+
+        for (int i = 0; i < BLOCKS.length; i++) if (BLOCKS[i] != null) BLOCK_MANAGER.registerCustomBlock(BLOCKS[i]);
+    }
```

Comment:
You access Minestom managers in a static block, you face the risk of accidentally accessing managers before they are initialized (in MinecraftServer#init() ).